pom.xml

```diff
@@ -1821,6 +1821,7 @@
     <xz.version>1.9</xz.version>
     <zstd-jni.version>1.5.0-4</zstd-jni.version>
     <hbase-thirdparty.version>4.0.1</hbase-thirdparty.version>
+    <nimbusds.version>9.15</nimbusds.version>
```

Comment:
Looks like they have 9.15.2 also released. I assume we would want to use the latest version.